packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -244,6 +244,61 @@ mixin RenderConstrainedLayoutBuilder<ConstraintType extends Constraints, ChildTy
 /// in an [Align] widget. If the child might want to be bigger, consider
 /// wrapping it in a [SingleChildScrollView] or [OverflowBox].
 ///
+/// {@tool snippet}
+///
+/// This example uses a [LayoutBuilder] to build a different widget depending on the available width.
```

Comment:
If you choose dartpad:
```suggestion
/// This example uses a [LayoutBuilder] to build a different widget depending on the available width. Resize the
/// DartPad window to see [LayoutBuilder] in action! 
```